src/pages/cli/auth/admin.mdx

```diff
@@ -263,4 +263,52 @@ class MyCustomInterceptor: URLRequestInterceptor {
 }
 ```
 </Block>
-</BlockSwitcher>
\ No newline at end of file
+</BlockSwitcher>
+
+## Adding Admin Actions
+
+To add additional admin actions that are not included by default but are enabled by Amazon Cognito, you will need to update the Lambda function code that is generated for you. The change will include adding a procedure for the action and creating a route for it. You will then associate the procedure to the route within the express app.
```

Comment:
```suggestion
To add additional admin actions that are not included by default but are enabled by Amazon Cognito, you will need to update the Lambda function code that is generated for you. The change will include adding a procedure for the action and creating a route for it. You will then associate the procedure to the route within the [Express](https://expressjs.com/) app.
```

minor suggestion since "express" is not mentioned in the page before this paragraph